src/main/fc/fc_msp.c

```diff
@@ -1482,6 +1482,9 @@ static mspResult_e mspFcProcessInCommand(uint8_t cmdMSP, sbuf_t *src)
         motorConfig()->motorPwmProtocol = constrain(sbufReadU8(src), 0, PWM_TYPE_BRUSHED);
 #endif
         motorConfig()->motorPwmRate = sbufReadU16(src);
+        if (dataSize > 7) {
```

Comment:
There are multiple occurences of `dataSize` on the same screen, but I like `sbufBytesRemaining` approach more because it does not accumulate the number of bytes read up to this point.